GetNOTAM.py

```diff
@@ -75,24 +79,38 @@ def buildNotam(effectiveStartDate, effectiveEndDate, long, lat, combined_core_no
 
 #runNotam: takes user input and does multiple buildNotam calls along a path in order 
 def runNotam():
-    effectiveStartDate, effectiveEndDate, long, lat, fLong, fLat = startNotam()
+    effectiveStartDate, effectiveEndDate, locations = startNotam()
     
     combined_core_notam_data = []
-    
-    combined_core_notam_data = buildNotam(effectiveStartDate, effectiveEndDate, long, lat, combined_core_notam_data)
-    print(len(combined_core_notam_data))
-    wlong = float(long)
-    wlat = float(lat)
-    
-    while wlong >= fLong or wlat <= fLat:
-        combined_core_notam_data = buildNotam(effectiveStartDate, effectiveEndDate, str(wlong), str(wlat), combined_core_notam_data)
-        if wlong >= fLong:
-            wlong = wlong - 0.16667
-        if wlat <= fLat:
-            wlat = wlat + 0.16667
-        print(wlat)
-        print(wlong)
+    for i, location in enumerate(locations[:-1]):
+        print(location)
+        airLocation = AirportsLatLongConverter.get_lat_and_lon(location)
+        finalAirLocation = AirportsLatLongConverter.get_lat_and_lon(locations[i+1])
+        long = airLocation[1]
+        lat = airLocation[0]
+        print(lat)
+        print(long)

```

Comment:
Presumably these `prints` here and below (and above) are for testing or debugging purposes.  Are we planning to remove them before merging?  If not, can we at least convert them to something more helpful, e.g.,
```python
print( f"Converted airport {location} into latitude, longitude of {lat}, {long}" )
```